src/D_EcobeeHouse1.json

```diff
@@ -35,10 +35,10 @@
     ],
     "inScene": "1",
     "DisplayStatus": {
-        "Service": "urn:upnp-org:serviceId:SwitchPower1",
-        "Variable": "Status",
-        "MinValue": "0",
-        "MaxValue": "1"
+        "Service": "urn:ecobee-com:serviceId:Ecobee1",
+        "Variable": "currentClimateRef",
+        "MinValue": "home",
+        "MaxValue": "smart1"
```

Comment:
I was wondering about this too when I wrote it. However it seems to work on UI7. The MinValue/MaxValue values don't appear to be needed indeed.